dolphinscheduler-api/pom.xml

```diff
@@ -63,13 +63,19 @@
 
         <dependency>
             <groupId>org.apache.dolphinscheduler</groupId>
-            <artifactId>dolphinscheduler-datasource-all</artifactId>
+            <artifactId>dolphinscheduler-common</artifactId>
```

Comment:
It's better to change `dolphinscheduler-datasource-all` to provided rather than remove it, once we remove it, no one can find the module relation.

Reply:
Fixed.